src/control/mod.rs

```diff
@@ -1067,10 +1072,36 @@ impl std::fmt::Debug for Mode {
             .field("hskew", &self.hskew())
             .field("vscan", &self.vscan())
             .field("vrefresh", &self.vrefresh())
+            .field("mode_type", &self.mode_type())
             .finish()
     }
 }
 
+/// Display mode type flags
+#[repr(u32)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum ModeType {
+    /// Deprecated builtin mode type
+    #[deprecated]
+    Builtin = ffi::DRM_MODE_TYPE_BUILTIN,
+    /// Deprecated CLOCK_C mode type
+    #[deprecated]
+    ClockC = ffi::DRM_MODE_TYPE_CLOCK_C,
+    /// Deprecated CRTC_C mode type
+    #[deprecated]
+    CrtcC = ffi::DRM_MODE_TYPE_CRTC_C,
+    /// Preferred mode
+    Preferred = ffi::DRM_MODE_TYPE_PREFERRED,
+    /// Deprecated default mode
+    Default = ffi::DRM_MODE_TYPE_DEFAULT,
```

Comment:
```suggestion
    /// Deprecated default mode
    #[deprecated]
    Default = ffi::DRM_MODE_TYPE_DEFAULT,
```
?

Nit, fwiw: I don't think we need to repeat the word `Deprecated` in the docs, as that's already covered by the `#[deprecated]` annotation:

![image](https://user-images.githubusercontent.com/2325264/136070876-e1f51224-a11f-4146-b9b5-4eeaee724a6d.png)